src/Catalog/NuGet.Services.Metadata.Catalog.csproj

```diff
@@ -300,6 +300,9 @@
     <PackageReference Include="NuGet.Services.Logging">
       <Version>2.25.0-master-30088</Version>
     </PackageReference>
+    <PackageReference Include="NuGetGallery.Core">
```

Comment:
why do we need this dependency? If it's only for CoreConstants.Sha512HashAlgorithmId lets just copy the constant. Extra dependencies create maintenance overhead.

Reply:
Yeah, you are right. The only usage of this dependency is "CoreConstants" for the key name. If you think hard coded key name is fine, I will use it here.